integrations/FacebookPixel/browser.js

```diff
@@ -101,6 +106,9 @@ class FacebookPixel {
         ];
         // this construcPayload will help to map the traits in the same way as cloud mode
         payload = constructPayload(rudderElement.message, traitsMapper);
+        if (payload.external_id) {
+          payload.external_id = sha256(payload.external_id);
```

Comment:
Should we add backward compatibility for this? @saikumarrs @ItsSudip

Reply:
I don't think we need backward compatibility here.